tpdatasrc/tpgamefiles/scr/spell_utils.py

```diff
@@ -125,6 +150,36 @@ def skillCheck(attachee, skillEnum, skillCheckDc):
     checkResult = True if skillRollResult >= skillCheckDc else False
     return checkResult
 
+#Check if a spell that remove curses is cast
+#This list can be expanded if new curse removal spells are added
+#Limited Wish, Miracle and Wish also remove Curses but are not added to the game
+def checkCurseRemoval(evt_obj):
+    if (evt_obj.is_modifier("sp-Break Enchantment")
```

Comment:
I absolutely want to avoid this kind of thing. I'd prefer modifiers that have this property register themselves.